changelogs/fragments/2059-Github-sanity-2-18-fix.yml

```
@@ -0,0 +1,4 @@
+trivial:
+  - ac-ansible-test-sanity.yml - Updated the ansible version to 2.18,
+    venv to 2.18, python to 2.12.
```

```suggestion
    venv to 2.18, python to 3.12.
```
Although it probably doesn't really matter since it's a trivial change.